loopdb/store.go

```diff
@@ -112,9 +112,49 @@ var (
 	// value: concatenation of uint32 values [family, index].
 	keyLocatorKey = []byte("keylocator")
 
+	// senderInternalPubKeyKey is the key that stores the sender's internal
+	// public key which used when constructing the swap HTLC.
+	//
+	// path: loopInBucket/loopOutBucket -> swapBucket[hash]
+	//                                      -> senderInternalPubKeyKey
+	// value: serialized public key.
+	senderInternalPubKeyKey = []byte("sender-internal-pubkey")
+
+	// receiverInternalPubKeyKey is the key that stores the receiver's
+	// internal public key which is used when constructing the swap HTLC.
+	//
+	// path: loopInBucket/loopOutBucket -> swapBucket[hash]
+	//                                     -> receiverInternalPubKeyKey
+	// value: serialized public key.
+	receiverInternalPubKeyKey = []byte("receiver-internal-pubkey")
+
+	// senderInternalKeyKey is the key that stores the sender's internal
+	// private key corresponding to the sender's internal public key.
+	//
+	// path: loopInBucket/loopOutBucket -> swapBucket[hash]
+	//                                     -> senderInternalKeyKey
+	// value: serialized private key.
+	senderInternalKeyKey = []byte("sender-internal-key")
```

Comment:
nit: rename to `senderInternalPrivKeyKey`

Reply:
done